sys/net/gnrc/netapi/gnrc_netapi.c

```diff
@@ -91,20 +93,20 @@ int gnrc_netapi_dispatch(gnrc_nettype_t type, uint32_t demux_ctx,
 
         while (sendto) {
 #if defined(MODULE_GNRC_NETAPI_MBOX) || defined(MODULE_GNRC_NETAPI_CALLBACKS)
-            int release = 0;
+            uint32_t errno = 0;
```

Comment:
gcc doesn't like this variable name (for obvious reason) :-)

Reply:
Fixed.